### ISSUE TITLE
I can add myself to a friends list

### ISSUE THREAD

**Issue by Serhii-Kaushnian** [ed-roh/mern-social-media#19]

**Comment by kotaid** [comment]
yeah, you can a check that the user is not the logged in user and not show the icons, also in the backend you can add that check.

**Comment by MitraPandit** [comment]
Just check if the _id matches with friendId in Friend.jsx. If yes, then dont show the icon, else show them
